packages/coachmark/test/coachmark-popover.test.ts

```diff
@@ -0,0 +1,216 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import {
+    elementUpdated,
+    expect,
+    fixture,
+    html,
+    oneEvent,
+} from '@open-wc/testing';
+
+import '@spectrum-web-components/coachmark/sp-coachmark-popover.js';
+import '@spectrum-web-components/coachmark/sp-coachmark-popover-content.js';
+import '@spectrum-web-components/button/sp-button.js';
+import {
+    Coachmark,
+    CoachmarkPopover,
+    CoachmarkPopoverContent,
+} from '@spectrum-web-components/coachmark';
+import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
+import { TemplateResult } from 'lit';
+import { Button } from '@spectrum-web-components/button';
+
+function isVisible(element: HTMLElement): boolean {
+    return !!element.offsetParent;
+}
+
+function template(open = false): TemplateResult {
+    return html`
+        <sp-coachmark-popover ?open=${open}>
+            <sp-button>Hello world</sp-button>
+            <sp-coachmark-popover-content
+                slot="coachmark"
+                heading="Learn about the world"
+                content="The world has some explaining to do!"
+                secondary-cta="Cancel"
+                primary-cta="Got it!"
+            ></sp-coachmark-popover-content>
+        </sp-coachmark-popover>
+    `;
+}
+
+describe('Coachmark Popover', () => {
+    testForLitDevWarnings(
+        async () => await fixture<CoachmarkPopover>(template(true))
+    );
```

Comment:
For some reason this test is giving me the following error:
```
TypeError: Attempted to wrap warn which is already wrapped
```

Reply:
I can't figure out why this is doing this, or why I can't cause other things to do this, or how I specifically prevented this when I saw it in the past, _BUT_ it seems that this patch makes it go away...

[controllerWrap.patch](https://github.com/adobe/spectrum-web-components/files/11245228/controllerWrap.patch)

Can you try this locally to confirm? I'd commit it separately, but I'm not a fan of commits that I can't test, and as I can't cause this error in other packages.